cli/command/service/opts.go

```diff
@@ -377,6 +403,7 @@ func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 			ContainerSpec: swarm.ContainerSpec{
 				Image:    opts.image,
 				Args:     opts.args,
+				Command:  opts.entrypoint.Value(),
```

Comment:
I believe that's correct. The value is set to `[]string(nil)`, which I guess is `== nil` in go.